dev-prefs.json

```diff
@@ -2,6 +2,8 @@
   "dom.ipc.processCount": 1,
   "javascript.options.strict": false,
   "xpinstall.signatures.required": false,
+  "extensions.logging.enabled": true,
   "extensions.@activity-streams.sdk.console.logLevel": "info",
-  "extensions.@activity-streams.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
+  "extensions.@activity-streams.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream",
+  "extensions.@activity-stream-shield-study.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
```

Comment:
Yes, it shares the same endpoint with the test pilot addon. Each payload of Shield has a "shield_variant" field to identify itself as a shield specific ping. So, we don't need a separate pref here for the endpoint.